bbrconn_linux.go

```diff
@@ -9,8 +9,13 @@ import (
 	"sync/atomic"
 
 	"github.com/getlantern/netx"
-	"github.com/getlantern/tcpinfo"
 	"github.com/mikioh/tcp"
+	"github.com/mikioh/tcpinfo"
+)
+
+const (
+	sizeOfTCPInfo    = 0xc0
+	sizeOfTCPBBRInfo = 0x14
```

Comment:
Are they taken from `zsys_linux.go`? It may worth a comment, esp. considering that future versions of `tcpinfo` may increase the size.